.github/workflows/test.yml

```diff
@@ -59,7 +47,7 @@ jobs:
           -J-Xmx3g
           -Denv.type=test
           -Dmysql.host=localhost
-          -Dmysql.port=3306
+          -Dmysql.port=3310
```

Comment:
`docker/run-mysql.sh` uses 3310 instead of 3306, therefore this change

Reply:
ahh I think this explains my other comment?

Reply:
yes, I thought it was better to update the GHA tests to use the same port as `docker/run-mysql.sh` instead of the other way around, because the change is more isolated